.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "python.testing.pytestArgs": [
+        "py"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.linting.enabled": true,
+	"python.linting.pylintEnabled": true
```

Comment:
@ethanrublee adding this seems to work fine in vscode but still not recognized during the pre-commit stage

Reply:
"pm Grant by4a.setedit22 android.permission.WRITE_SECURE_SETTINGS".